realm/realm-library/src/objectServer/java/io/realm/RealmFileUserStore.java

```diff
@@ -74,17 +80,27 @@ public void remove() {
         return Collections.emptyList();
     }
 
+    private static SyncUser toSyncUserOrNull(String userJson) {
+        if (userJson == null) {
+            return null;
+        }
+        return SyncUser.fromJson(userJson);
+    }
+
     // init and load the Metadata Realm containing SyncUsers
```

Comment:
Although it is not part of your PR, should `init and load` be `inits and loads`?